internal/getproviders/package_authentication_helper_test.go

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) The OpenTofu Authors
+// SPDX-License-Identifier: MPL-2.0
+// Copyright (c) 2023 HashiCorp, Inc.
```

Comment:
`question`
Unrelated to the actual changes, just want to clarify this.
For new files, do we need to add the HC copyright headers too?

Reply:
We err on the side of caution, given that it is easy for code to be moved around in the codebase.